### PR TITLE
11.0 mig l10n nl dutch company type

### PR DESCRIPTION
**Pull request by lfreeke**

### CODE REVIEW

**Comment by hbrunn** [comment]
please fix https://travis-ci.org/OCA/l10n-netherlands/jobs/458460707#L568

this depends on `partner_company_type` which isn't yet migrated in v11

**Comment by lfreeke** [comment]
@hbrunn  https://github.com/OCA/partner-contact/pull/660 there is a pull request for partner_company_type. Should I wait until that is merged in?

**Comment by hbrunn** [comment]
yes, we can only go further here if all dependencies are merged

File: l10n_nl_dutch_company_type/__init__.py
Comment (NL66278) [inline l10n_nl_dutch_company_type/__init__.py]: Do away with Copyright line from __init__.py files

File: l10n_nl_dutch_company_type/__manifest__.py
Comment (NL66278) [inline l10n_nl_dutch_company_type/__manifest__.py:9]: Add website key with url https://github.com/OCA/l10n-netherlands

File: l10n_nl_dutch_company_type/readme/CREDITS.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/CREDITS.rst]: Remove this file and then regenerate README.rst

File: l10n_nl_dutch_company_type/readme/DESCRIPTION.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/DESCRIPTION.rst]: Give a real description here.

File: l10n_nl_dutch_company_type/readme/HISTORY.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/HISTORY.rst]: get rid of this file as well

File: l10n_nl_dutch_company_type/readme/ROADMAP.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/ROADMAP.rst]: And get rid of this one

File: l10n_nl_dutch_company_type/readme/USAGE.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/USAGE.rst]: And this one.

**Review by NL66278 — CHANGES_REQUESTED** [review]
See comments

**Comment by lfreeke** [comment]
> yes, we can only go further here if all dependencies are merged

The dependency is merged. https://github.com/OCA/partner-contact/pull/660

**Comment by NL66278** [comment]
@hbrunn Can you remove the requisite branding label?

File: l10n_nl_dutch_company_type/__manifest__.py
Comment (NL66278) [inline l10n_nl_dutch_company_type/__manifest__.py]: make it https

File: l10n_nl_dutch_company_type/readme/INSTALL.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/INSTALL.rst]: Please remove this file (and regenerate README)

File: l10n_nl_dutch_company_type/readme/DESCRIPTION.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/DESCRIPTION.rst]: Put some real description here. Maybe a link to a wiki or to  standard where the dutch company types are described.

File: l10n_nl_dutch_company_type/readme/CONFIGURE.rst
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/CONFIGURE.rst]: Get rid of the first three lines.

**Comment by lfreeke** [comment]
@NL66278 I resolved the comments.

File: l10n_nl_dutch_company_type/__manifest__.py
Comment (NL66278) [reply l10n_nl_dutch_company_type/__manifest__.py]: But license is still http?

Comment (lfreeke) [reply l10n_nl_dutch_company_type/__manifest__.py]: Now it is https

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM + tested

File: l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot:7]: 11

File: l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot:19]: no translation?

File: l10n_nl_dutch_company_type/i18n/nl.po
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/nl.po]: 11

File: l10n_nl_dutch_company_type/i18n/nl.po
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/nl.po:11]: add name

**Review by gfcapalbo — CHANGES_REQUESTED** [review]

File: l10n_nl_dutch_company_type/README.rst
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/README.rst:38]: Delete or complete readme

File: l10n_nl_dutch_company_type/README.rst
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/README.rst]: real name

File: l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot
Comment (lfreeke) [reply l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot:19]: No translation needed here, it is the pot file, translation are in nl.po

**Comment by lfreeke** [comment]
Is there a possibility that someone can rebuild this branch? Travis is giving this error 

psycopg2.OperationalError: could not connect to server: No such file or directory
Is the server running locally and accepting
connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?

**Comment by NL66278** [comment]
@lfreeke Update (in a separate commit) .travis.yml. Use 11.0 version of partner-contact as a guideline. At least update postgres to 9.6 instead of 9.2. Also add a MAKEPOT="0" to the line under global:

**Comment by NL66278** [comment]
@lfreeke Ping

**Comment by lfreeke** [comment]
@NL66278 see this merge request https://github.com/OCA/l10n-netherlands/pull/257 and this one https://github.com/OCA/l10n-netherlands/pull/258

**Pull request merged**
